core/templates/dev/head/editor/ExplorationEditor.js

```diff
@@ -759,6 +761,14 @@ oppia.controller('ExplorationSaveAndPublishButtons', [
           }, function() {
             $scope.isSaveInProgress = false;
           });
+
+          if ($scope.isPrivate()) {
+            siteAnalyticsService.registerSaveExplorationDraftEvent(
```

Comment:
That's not unreasonable, though I think the current names are more semantic (and I wanted to avoid over-coupling things to the private/public status). Do you feel strongly?